spec/consensus/light-client.md

```diff
@@ -6,45 +6,46 @@ A lite client is a process that connects to Tendermint full nodes and then tries
 
 In order to make sure that full nodes have the incentive to follow the protocol, we have to address the following three Issues
 
-1. The lite client needs a method to verify headers it obtains from full nodes according to trust assumptions -- this document.
+1) The lite client needs a method to verify headers it obtains from a full node it connects to according to trust assumptions -- this document.
 
-2. The lite client must be able to connect to one correct full node to detect and report on failures in the trust assumptions (i.e., conflicting headers) -- a future document.
+2) The lite client must be able to connect to other full nodes to detect and report on failures in the trust assumptions (i.e., conflicting headers) -- a future document (see #4215).
 
-3. In the event the trust assumption fails (i.e., a lite client is fooled by a conflicting header), the Tendermint fork accountability protocol must account for the evidence -- see #3840
+3) In the event the trust assumption fails (i.e., a lite client is fooled by a conflicting header), the Tendermint fork accountability protocol must account for the evidence -- a future document (see #3840).
 
 ## Problem statement
 
-We assume that the lite client knows a (base) header _inithead_ it trusts (by social consensus or because the lite client has decided to trust the header before). The goal is to check whether another header _newhead_ can be trusted based on the data in _inithead_.
 
-The correctness of the protocol is based on the assumption that _inithead_ was generated by an instance of Tendermint consensus. The term "trusting" above indicates that the correctness on the protocol depends on this assumption. It is in the responsibility of the user that runs the lite client to make sure that the risk of trusting a corrupted/forged _inithead_ is negligible.
+We assume that the lite client knows a (base) header *inithead* it trusts (by social consensus or because the lite client has decided to trust the header before). The goal is to check whether another header *newhead* can be trusted based on the data in *inithead*.
+
+The correctness of the protocol is based on the assumption that *inithead* was generated by an instance of Tendermint consensus. The term "trusting" above indicates that the correctness of the protocol depends on this assumption. It is in the responsibility of the user that runs the lite client to make sure that the risk of trusting a corrupted/forged *inithead* is negligible.
+
 
 ## Definitions
 
 ### Data structures
 
 In the following, only the details of the data structures needed for this specification are given.
 
-- header fields
+  * header fields
+    - *height*
+    - *bfttime*: the chain time when the header (block) was generated
+    - *V*: validator set containing validators for this block.
+    - *NextV*: validator set for next block.
```

Comment:
Does this mean, the `ValidatorSet` will be expected to be explicitly contained in the header now? Probably not but it is used as a field below (e.g. in `verify`: `votingpower_in(signers(h.Commit),h.Header.V)`)?

Reply:
Good point. We should be more precise with respect to this. The assumption was that having hash of the header you can fetch the corresponding validator set. But I agree that it's better being precise there and having explicit rpc call as it might turn out that it make sense adding this information to SignedHeader (for example) and the fetching it with `Commit` rpc call.